src/metrics/types/index.js

```diff
@@ -24,9 +24,29 @@ const Types = {
  */
 function getByName(name) {
 	/* istanbul ignore next */
-	if (!name) return null;
+	if (name === undefined) return null;
```

Comment:
Same problem, why `name == null` is valid?

Reply:
Again, probably because of numbers? 0?